pkg/plugin/plugin.go

```diff
@@ -102,8 +101,8 @@ func (p *plugin) GenerateARM(ctx context.Context, cs *api.OpenShiftManagedCluste
 
 func (p *plugin) InitializeCluster(ctx context.Context, cs *api.OpenShiftManagedCluster) error {
 	log.Info("initializing cluster")
-	initializer := initialize.NewSimpleInitializer(p.entry, p.config)
-	return initializer.InitializeCluster(ctx, cs)
+	upgrader := upgrade.NewSimpleUpgrader(p.entry, p.config)
+	return upgrader.InitializeCluster(ctx, cs)
```

Comment:
Maybe just a semantics but seeing `Upgrader` doing `InitializeCluster` does not look right. Dunno :)

Reply:
Yeah, I think the upgrade package should be called "cluster"
cluster.Initialize()
cluster.Update()
cluster.Deploy() etc..
because we have a config package with a upgrader as well..

Reply:
Im up for this rename :)